project.clj

```diff
@@ -11,6 +11,8 @@
                  [org.clojure/clojurescript "1.7.122"]
                  [org.clojure/core.async "0.2.374"]]
 
+  :plugins [[lein-cljfmt "0.3.0"]]
```

Comment:
Is there a difference b/t adding the plugin here vs. in `-> :profiles :dev :plugins`?